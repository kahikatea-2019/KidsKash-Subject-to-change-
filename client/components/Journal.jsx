import React from 'react'

function Journal () {
  return (
    <h2>Over here as a parent I can make comments and journal so I can be accountable for my behaviour and pick up on trends on my child behaviour</h2>
  )
}

export default Journal
