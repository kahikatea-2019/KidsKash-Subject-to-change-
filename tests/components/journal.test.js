import React from 'react'
import { shallow } from 'enzyme'
import Journal from '../../client/components/Journal'

const journal = shallow(<Journal />)

test('test <Journal /> is set up correctly', () => {
  expect(true).toBe(true)
})

test('<Journal/> has 1 h2 tag', () => {
  expect(journal.find('h2').length).toBe(1)
})

describe('journal', () => {
  it('returns correctly', () => {
    expect(journal).toMatchSnapshot()
  })
})