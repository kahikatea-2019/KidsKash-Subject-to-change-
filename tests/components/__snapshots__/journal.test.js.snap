// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`journal returns correctly 1`] = `ShallowWrapper {}`;
